src/components/victory-axis/helper-methods.js

```diff
@@ -114,5 +114,17 @@ module.exports = {
       right: [props.width - offset.x, 0]
     }[orientation];
     return `translate(${translate[0]}, ${translate[1]})`;
+  },
+
+  getTickPosition(style, orientation, isVertical) {
+    const orientationSign = { top: -1, left: -1, right: 1, bottom: 1 };
+    const tickSpacing = style.size + style.padding;
+    const sign = orientationSign[orientation];
+    return {
+      x: isVertical ? sign * tickSpacing : 0,
+      x2: isVertical ? sign * style.size : 0,
```

Comment:
What are the `x2` and `y2` properties? I can see that they don't include padding. How are these positions meant to be used?